### ISSUE TITLE
Temperature: Wrong value displayed in HomeKit

### ISSUE THREAD

**Issue by juergen4mueller** [dehsgr/homebridge-mobilealerts#2]
I use the homebridge-mobilealerts plugin.
Most of my sensors are not supported.
I get right information about humidity, but temperature os always displayed with 0°.

Would be great if you find time to implement more sensors.
I hae wind and rain sensor too, unluckily not working with this great plugin.

Here the response from my system:

'
<h3>&#220;berblick f&#252;r Phone ID xxxxxxxx372</h3>

<p><a href="/">Zur&#252;ck zu Home</a></p>

<div class="panel panel-default">
    <div class="panel-body">
        <div class="sensor">

<div class="sensor-header">
    <h3>
        <a href="/Home/MeasurementDetails?deviceid=084B0BF3xxxx&amp;vendorid=455E2A8E-E711-4D09-A5E8-2525E7E25EC7&amp;appbundle=de.synertronixx.remotemonitor">Niederschlag</a>
    </h3>
    <div class="sensor-component">
        <h5>ID</h5>
        <h4>084B0BF3B58B</h4>
    </div>
</div>

<div class="nofloat"></div>

    <div class="sensor-component">
        <h5>Zeitpunkt</h5>
        <h4>05.02.2018 14:25:52</h4>
    </div>
    <div class="sensor-component">
        <h5>Regen</h5>
        <h4>0,0 mm</h4>
    </div>
        </div>
    </div>
</div>

<div class="panel panel-default">
    <div class="panel-body">
        <div class="sensor">

<div class="sensor-header">
    <h3>
        <a href="/Home/MeasurementDetails?deviceid=0312AE01xxxx&amp;vendorid=455E2A8E-E711-4D09-A5E8-2525E7E25EC7&amp;appbundle=de.synertronixx.remotemonitor">Temperatur</a>
    </h3>
    <div class="sensor-component">
        <h5>ID</h5>
        <h4>0312AE012D8A</h4>
    </div>
</div>

<div class="nofloat"></div>

    <div class="sensor-component">
        <h5>Zeitpunkt</h5>
        <h4>09.02.2018 14:13:15</h4>
    </div>
    <div class="sensor-component">
        <h5>Temperatur</h5>
        <h4>0,0 C</h4>
    </div>
    <div class="sensor-component">
        <h5>Luftfeuchtigkeit</h5> 
        <h4>84%</h4>
    </div>
        </div>
    </div>
</div>

<div class="panel panel-default">
    <div class="panel-body">
        <div class="sensor">

<div class="sensor-header">
    <h3>
        <a href="/Home/MeasurementDetails?deviceid=0B020029xxxx&amp;vendorid=455E2A8E-E711-4D09-A5E8-2525E7E25EC7&amp;appbundle=de.synertronixx.remotemonitor">Wind </a>
    </h3>
    <div class="sensor-component">
        <h5>ID</h5>
        <h4>0B0200296452</h4>
    </div>
</div>

<div class="nofloat"></div>

    <div class="sensor-component">
        <h5>Zeitpunkt</h5>
        <h4>09.02.2018 14:11:43</h4>
    </div>
    <div class="sensor-component">
        <h5>Windgeschwindigkeit</h5>
        <h4>2,5 km/h</h4>
    </div>
    <div class="sensor-component">
        <h5>B&#246;e</h5>
        <h4>5,8 km/h</h4>
    </div>
    <div class="sensor-component">
        <h5>Windrichtung</h5>
        <h4>Südwest</h4>
    </div>
        </div>
    </div>
</div>
'

**Comment by svobluk** [comment]
the zero values inside the homekit are bcos homekit doesn't allow negative temperature. It has to be implemented to the script (during accessory initialization) or you have to change minimal temp. value, see this thread -> https://github.com/KhaosT/HAP-NodeJS/issues/147#issuecomment-147234891

**Comment by dehsgr** [comment]
@juergen4mueller:
Negative temperature is fixed now. I also took a look on your sensors. MA10200 was already supported in previous commit. Did you update the plugin via npm and restarted homebridge? I verified that there was shown temperature and humidity. The other sensors currently are not supported by homekit. If you find a solution where it is already working, I'll take a closer look again. I verified with https://github.com/naofireblade/homebridge-weather-station-extended and there were no usable data regardless of eve or Apple's Home app.

@svobluk:
Thanks! You did it for me. ;-)
